libs/designer/src/lib/ui/panel/panelTabs/parametersTab/index.tsx

```diff
@@ -90,7 +98,7 @@ export const ParametersTab = () => {
           {errorInfo.message}
         </MessageBar>
       ) : null}
-      {!hasParametersToAuthor(inputs?.parameterGroups ?? {}) ? (
+      {!errorInfo && !hasParametersToAuthor(inputs?.parameterGroups ?? {}) ? (
```

Comment:
We should not be checking for warning errors, for example dynamic outputs load error should not impact inputs loading

Reply:
This block doesn't impact the actual inputs, it's just the message bar that appears when there are no inputs, but I get what you're saying and I can adjust to be more specific in the errors that trigger it.  I mainly was trying to catch when we have errors around getting dynamic inputs and possibly being empty as an error state instead of intentionally